### ISSUE TITLE
Filter by recipient names on feedback responses page (S)

### ISSUE THREAD

**Issue by pieperm** [oci-labs/check-ins#1334]
Depends on #1271 

## Definition of Done
- [x] Search bar on feedback responses page
- [x] Search bar displays a chip for each member whose responses should be shown
- [x] If no chips are in the search bar, all responses should be shown (from whatever the original entry point was)
- [x] Autocomplete in search field for selecting members (*stretch goal*)
